### PR TITLE
docs: add feedback form to k8s update page

### PR DESCRIPTION
**Pull request by ggilmore**
Fixes: #18031 

- Adds a link to a Hubspot feedback form created by @ebrodymoore that solicits feedback from site admins 
	- This link will be added to https://github.com/sourcegraph/sourcegraph/blob/main/doc/admin/updates/kubernetes.md. Our customer interviews showed us that customers keep this page open throughout the upgrade process, so it seems like a contextually relevant place to ask for feedback there. 
- Changes our release tooling to insert a link to the feedback form in the upgrade guide for each new release

Known issues:

- The hubspot feedback form doesn't have any Sourcegraph branding on it. It also uses a "raw" Hubspot URL. I am told that our marketing team can add both of these things later on, but it doesn't seem like a blocker to me.

- The hubspot page isn't linked up to product board as is our preference. I don't think this is a blocker since we can look into hooking this up after the fact. 

---

The Hubspot page for the form can be viewed here (requires authentication): https://app.hubspot.com/forms/2762526/editor/686786ec-02d0-4041-8eeb-3c9f6ae22208/edit/form

### CODE REVIEW

File: dev/release/src/util.ts [hunk 1]
```diff
@@ -56,3 +57,15 @@ export function getWeekNumber(date: Date): number {
...
 }
+
+export function hubSpotFeedbackFormStub(version: string): string {
+    const link = `[this feedback form](${hubSpotFeedbackFormURL(version)})`
+    return `*How smooth was this upgrade process for you? You can give us your feedback on this upgrade by filling out ${link}.*`
+}
+
+function hubSpotFeedbackFormURL(version: string): string {
+    const url = new URL('https://share.hsforms.com/1aGeG7ALQQEGO6zyfauIiCA1n7ku')
+    url.searchParams.set('update_version', version)
```
Comment (ggilmore) [inline dev/release/src/util.ts:68]: using the URL class here isn't strictly necessary, but it seems more robust to me (as opposed to just gluing the string together

**Comment by daxmc99** [comment]
Should we ask a frontend-dev to help review the code here? I'm not familiar enough with our frontend code to review that portion.

**Comment by ggilmore** [comment]
@daxmc99 What frontend code are you referring to? The changes I made were to our docs page and release tool I believe.

**Comment by daxmc99** [comment]
Ahh I saw typescript and I thought it was adding the frontend code

**Review by pecigonzalo — APPROVED** [review]
LGTM, please rebase and add 3.26 before merging @ggilmore

**Pull request merged**